src/theme/components/multiSelect.ts

```diff
@@ -0,0 +1,97 @@
+const multiSelectWidths = {
+  default: {
+    width: {
+      base: "100%",
+      md: "250px",
+    },
+  },
+  fitContent: {
+    width: {
+      base: "100%",
+      md: "fit-content",
+    },
+    minWidth: {
+      md: "175px",
+    },
+  },
+  full: {
+    width: "100%",
+  },
+};
+
+const MultiSelect = {
+  parts: [
+    "menuButton",
+    "menuContainer",
+    "menu",
+    "menuChildren",
+    "selectedItemsCountButton",
+  ],
+  baseStyle: ({ width = "default", isBlockElement }) => ({
+    ...multiSelectWidths[width],
+    position: "relative",
+    margin: {
+      base: ".75rem 0",
+      md: "0 2em 0 0",
+    },
+    menuButton: {
+      width: "100%",
+      justifyContent: "space-between",
+    },
+    menuContainer: {
+      position: {
+        md: isBlockElement ? "relative" : "absolute",
+      },
+      zIndex: isBlockElement ? 0 : 1,
```

Comment:
Just to be safe, make this `? 0 : 9999`. If I understand this correctly, this makes the menu hover over elements when `isBlockElement` is false. What we don't know is if the content under the menu has a high z-index. If it does, then "1" will not be enough. This ensures that the open menu will hover over content.

Reply:
fixed

Reply:
fixed